ceph-dashboard-cephadm-e2e-nightly/config/definitions/ceph-dashboard-cephadm-e2e-nightly.yml

```diff
@@ -70,3 +70,9 @@
     publishers:
       - email:
           recipients: ceph-qa@ceph.io
+
+    publishers:
+      - archive:
```

Comment:
This should go on line 73.   There should not be multiple `publisher` sections.

Reply:
you are right! Done the changes now.